mlflow/gateway/constants.py

```diff
@@ -0,0 +1,6 @@
+GATEWAY_VERSION = "0.1.0"
+CONF_PATH_ENV_VAR = "MLFLOW_GATEWAY_CONFIG_PATH_ENV_VAR_NAME"
+LOCALHOST = "127.0.0.1"
+SERVER_HOUSEKEEPING_PAUSE = 2
```

Comment:
Can we add the units here?

Reply:
added to the new constants (removed this one)